pyproject.toml

```diff
@@ -52,3 +203,41 @@ force_alphabetical_sort_within_sections = true
 [tool.rstcheck]
 report_level = "warning"
 ignore_directives = ["automodule", "autoclass", "autofunction", "autodata"]
+
+[tool.liccheck]
+authorized_licenses = [
+  "bsd",
+  "new bsd",
+  "simplified bsd",
+  "apache",
+  "apache 2.0",
+  "apache software",
+  "Apache License 2.0",
+  "gnu lgpl",
+  "gpl v2",
+  "GNU General Public License v2 or later (GPLv2+)",
+  "GNU Library or Lesser General Public License (LGPL)",
+  "gpl v3",
+  "lgpl with exceptions or zpl",
+  "isc",
+  "isc license (iscl)",
+  "mit",
+  "mit license",
+  "python software foundation",
+  "zpl 2.1",
+  "mpl-2.0",
+  "MPL 2.0",
+  "Mozilla Public License 2.0 (MPL 2.0)",
+  "lgpl",
+  "BSD-like",
+  "Public Domain",
+  "The Unlicense (Unlicense)",
+	"Zope Public",
+]
+
+[tool.liccheck.authorized_packages]
+twisted = "23.8.0"
```

Comment:
Can't we add Twisted's licence to the list of authorized licenses instead ?

Reply:
The license of twisted is MIT, and it is in the license list. For some reason the liccheck was failing, that's why I added it to the authorized packages.

Reply:
Ah, I see that the Twisted metadata file has a `License-File` and a `License-Expression` tag, but no `License` tag, that's why Liccheck reports it as `UNKNOWN`.
Let's keep it that way, but when you have time could you propose a PR to liccheck to make it read `License-Expression` as well please?